xc/common/utils/prjxray_routing_import.py

```diff
@@ -441,7 +441,11 @@ def check_feature(feature):
         features.append(
             '{}.{}_ACTIVE'.format(feature_path[0], feature_path[-1])
         )
-        features.append('{}.{}_USED'.format(feature_path[0], feature_path[-1]))
+
+        if "FREQ_REF" not in feature_path[-2]:
+            features.append(
```

Comment:
Can you add a comment of the pattern you are matching here?  I think it will be helpful

Reply:
Right, added a comment explaining it.